`make dist` fails due to missing bzip header

```
checking for bzlib.h... no
real_configure: error: could not find bzlib.h - the header of libbz2 library. Cf. https://doc.sagemath.org/html/en/reference/spkg/_prereq.html
```
https://github.com/sagemath/sage/actions/runs/15888768081/job/44806968371#step:4:98

--- a/.github/workflows/ci-macos.yml
+++ b/.github/workflows/ci-macos.yml
@@ -87,7 +87,7 @@ jobs:
       - name: Install bootstrap prerequisites
         run: |
           sudo DEBIAN_FRONTEND=noninteractive apt-get update
-          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap)
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap _prereq)
       - name: Bootstrap with update-version
         # We set SAGE_ROOT and SAGE_SRC by hand
         # because 'sage -sh' does not work with an unconfigured tree,

--- a/.github/workflows/dist.yml
+++ b/.github/workflows/dist.yml
@@ -37,7 +37,7 @@ jobs:
       - name: Install bootstrap prerequisites
         run: |
           sudo DEBIAN_FRONTEND=noninteractive apt-get update
-          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap)
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap _prereq)
       - name: make dist (--disable-download-from-upstream-url)
         id: make_dist
         run: |
@@ -106,7 +106,7 @@ jobs:
       - name: Old sagemath-standard
         run: |
           sudo DEBIAN_FRONTEND=noninteractive apt-get update
-          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap)
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap _prereq)
           ./bootstrap
           ./configure
           make pypi-sdists V=0
@@ -187,7 +187,7 @@ jobs:
       - name: Install bootstrap prerequisites
         run: |
           sudo DEBIAN_FRONTEND=noninteractive apt-get update
-          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap)
+          sudo DEBIAN_FRONTEND=noninteractive apt-get install $(build/bin/sage-get-system-packages debian _bootstrap _prereq)
       - name: make pypi-noarch-wheels
         run: |
           ./bootstrap
@@ -266,7 +266,7 @@ jobs:
       - name: Bootstrap
         run: |
           export PATH=$(pwd)/build/bin:$PATH
-          eval $(sage-print-system-package-command auto --sudo --yes --no-install-recommends --spkg install _bootstrap)
+          eval $(sage-print-system-package-command auto --sudo --yes --no-install-recommends --spkg install _bootstrap _prereq)
           ./bootstrap
 
       - name: Unpack and prepare

